modules/json_content/json_content.routing.yml

```diff
@@ -0,0 +1,7 @@
+d8contrib.frontpage:
+  path: '/frontpage'
+  defaults:
+    _title: 'Login'
+    _controller: '\Drupal\json_content\Controller\FrontController:frontpage'
+  requirements:
+    _permission: 'access content'
```

Comment:
I think this should be accessible in all circumstances, so let's make this _access: 'TRUE'